source/api_documentation/network_integration/tts_prompt_voice/index.rst

```diff
@@ -0,0 +1,213 @@
+TTS Prompt Voice
+======================
+
+For full details on creating and updating campaigns via the API, see :doc:`Advertiser Campaigns <../advertiser_campaigns/index>`.
+
+Manage TTS Prompt Voice for Campaigns
+"""""""""""""""""""""""""""""""""""""""""""
+
+TTS prompt voice can be sent for a campaign to use different accents for TTS prompts on a call.
+
+Set TTS Prompt Voice
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To generate voice prompts using a certain prompt voice for a given campaign, set ``"tts_voice_talent_id"`` on the IVR you want.
+
+POST
+
+``https://invoca.net/api/@@CAMPAIGN_FEATURES_API_VERSION/<network_id>/advertisers/<advertiser_id_from_network>/advertiser_campaigns/<advertiser_campaign_id_from_network>.json``
+
+Example Request Body
+
+.. code-block:: json
+
+  {
+    "ivr_tree": {
+      "tts_voice_talent_id": 3,
+      "root": {
+        "node_type": "Connect",
+        "destination_phone_number": "8056173768",
+        "destination_country_code": ""
+      }
+    }
+  }
+
+Conditions
+~~~~~~~~~~
+
+English voice options
+---------------------
+
+.. list-table::
+  :widths: 10 10 10 10
+  :header-rows: 1
+  :class: parameters
+
+  * - Option
+    - Voice Name
+    - Gender
+    - Language code (Country)
+
+  * - 3 (Default)
+    - Julie
+    - Female
+    - en-US (United States)
+
+  * - 7
+    - Bridget
+    - Female
+    - en-UK (United Kingdom)
+
+  * - 8
+    - Nicole
+    - Female
+    - en-AU (Australia)
+
+  * - 9
+    - Russell
+    - Male
+    - en-AU (Australia)
+
+  * - 10
+    - Amy
+    - Female
+    - en-UK (United Kingdom)
+
+  * - 11
+    - Emma
+    - Female
+    - en-UK (United Kingdom)
+
+  * - 12
+    - Brian
+    - Male
+    - en-UK (United Kingdom)
+
+  * - 13
+    - Aditi
+    - Female
+    - en-IN (India)
+
+  * - 14
+    - Raveena
+    - Female
+    - en-IN (India)
+
+  * - 15
+    - Joanna
+    - Female
+    - en-US (United States)
+
+  * - 16
+    - Kendra
+    - Female
+    - en-US (United States)
+
+  * - 17
+    - Kimberly
+    - Female
+    - en-US (United States)
+
+  * - 18
+    - Salli
+    - Female
+    - en-US (United States)
+
+  * - 19
+    - Joey
+    - Male
+    - en-US (United States)
+
+  * - 20
+    - Matthew
+    - Male
+    - en-US (United States)
+
+  * - 21
+    - Geraint
+    - Male
+    - en-UK (United Kingdom)
+
+
+French voice options
+--------------------
+
+.. list-table::
+  :widths: 10 10 10 10
+  :header-rows: 1
+  :class: parameters
+
+  * - Option
```

Comment:
Same comment here

> Because the first column is actually the id of the voice, and maps to what the customer will need to use as the tts_voice_talent_id, I suggest we rename the column header Options to Id to make that clear.